presto-docs/src/main/sphinx/release/release-0.242.rst

```diff
@@ -0,0 +1,52 @@
+=============
+Release 0.242
+=============
+
+General Changes
+_______________
+* Fix compiler error in LambdaBytecodeGenerator when CSE is enabled and multiple SQL functions contain lambda expressions.
+* Fix possible permanent stuck queries when memory limit exceeded during shuffle.
+* Fix a bug where ``IS DISTINCT FROM NULL`` produces wrong results for columns with null values.
+* Add config ``experimental.spiller.task-spilling-strategy`` for choosing different spilling strategy to use.
+* Add session property ``query_max_output_bytes`` and configuration property ``query.max-output-bytes`` to control how much data a query can output.
+* Add support for file renaming for Hive connector. This can be enabled with ``hive.file_renaming_enabled`` configuration property.
+* Add support for fragment result caching. When enabled, if the same plan fragment and same connector split hit the same worker, engine would directly fetch result from cache and skip computation. Currently only partial aggregation is supported. Cache could be enabled by setting ``fragment-result-cache.enabled`` to ``true`` and tuned by other configs started with ``fragment-result-cache``. Query could use fragment result cache by setting config ``experimental.fragment-result-caching-enabled`` or session property ``fragment_result_caching_enabled`` to ``true``.
```

Comment:
Is there a separate rst file documenting how this could be used? This should not be explained in release notes. @shixuan-fan

Reply:
No we don't. I could work on the rst file.

How about we just say 
```
Add support for fragment result caching. This can be enabled by ``fragment-result-cache.enabled`` and ``experimental.fragment-result-caching-enabled``
```
In the meanwhile, we could work on rst file.